coala_quickstart/generation/SettingsClass.py

```diff
@@ -354,3 +354,8 @@ def collect_bear_settings(bears):
         for bear in bears[language]:
             bear_settings_obj.append(BearSettings(bear))
     return bear_settings_obj
+
+
+def build_bear_settings(bears):
+    pass  # pragma: no cover
+    # TODO build bear_settings.yaml on the fly when the bear args are annotated
```

Comment:
do you have a WIP for this, that you intend to get merged before the dead line?

if not, remove it.